rules/windows/defense_evasion_injection_msbuild.toml

```
@@ -33,8 +33,7 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where host.os.type == "windows" and process.name: "MSBuild.exe" and
-    event.action:("CreateRemoteThread detected (rule: CreateRemoteThread)", "CreateRemoteThread")
+process where host.os.type == "windows" and event.code == "8" and process.name: "MSBuild.exe"
```

to avoid hitting on event.code with similar values and unrelated to sysmon: 

```suggestion
process where host.os.type == "windows" and 
 event.dataset in ("windows.sysmon_operational", "windows.sysmon") and 
/* CreateRemoteThread */
event.code == "8" and process.name: "MSBuild.exe"
```